.circleci/config.base.yml

```diff
@@ -198,6 +198,7 @@ jobs:
             yarn yarn-use-bash
             yarn pkg-all
             unsetNpmRegistryUrl
+            uploadPkgCli
```

Comment:
the pkg-cli must be uploaded to our deployment s3 bucket in order for `npm install` to function properly, as a the postinstall hook downloads the binary.